deploy/crds/kieapp.crd.yaml

```diff
@@ -770,15 +770,6 @@ spec:
                     amqClusterPassword:
                       type: string
                       description: The password to use for amq cluster user.
-                    controllerPassword:
-                      type: string
-                      description: The password to use for the controllerUser.
-                    serverPassword:
-                      type: string
-                      description: The password to use for the executionUser.
-                    mavenPassword:
-                      type: string
-                      description: The password to use for the mavenUser.
```

Comment:
@ruromero @bmozaffa 
More a conversation starter than anything, should we version our api for adds/removes or only "changes" to existing components?

Reply:
I think we should decide case by case, with a major driver of our decision being the difficulty and complication of version increments, which I hope will get better. In fact the level of difficulty should even affect our decision to make CRD changes.

As long as CRD versioning remains as difficult as it has been, I would advocate to only version when there is a change, or required field is removed, or similar. New critical and required fields may be an exception, if we want to make sure they are exposed in the CRD.

Reply:
In this case the remove fields are not required and most of the times generated. I don't consider this a breaking change but that's something to always have in mind and agree on beforehand.
Thanks